test/tesselation.jl

```diff
@@ -45,5 +45,13 @@
     pts = randpoint3(10)
     pset = PointSet(pts)
     @test_throws AssertionError tesselate(pset, VoronoiTesselation(StableRNG(2024)))
+
+    # Test polygon order is the same as input points order
+    rng = StableRNG(2024)
+    pts = randpoint2(10)
+    mesh = tesselate(pts, VoronoiTesselation(rng))
```

Comment:
```suggestion
    pts = randpoint2(10)
    mesh = tesselate(pts, VoronoiTesselation(StableRNG(2024))
```

Sorry, didn't notice that other tests were using this pattern. Undoing it.